extensions/default/src/getHangingProtocolModule.js

```diff
@@ -1,188 +1,313 @@
-const hangingProtocolName = 'petCT';
+import ConfigPoint from 'config-point';
 
-const petCTProtocol = {
-  id: 'PET/CT',
-  locked: true,
-  hasUpdatedPriorsInformation: false,
-  name: 'PET/CT',
-  createdDate: '2021-02-23T18:32:42.849Z',
-  modifiedDate: '2021-02-23T18:32:42.849Z',
-  availableTo: {},
-  editableBy: {},
-  protocolMatchingRules: [
-    {
-      id: 'wauZK2QNEfDPwcAQo',
-      weight: 1,
-      attribute: 'StudyInstanceUID',
-      constraint: {
-        equals: {
-          value: '1.3.6.1.4.1.25403.345050719074.3824.20170125112931.11',
-        },
-      },
-      required: true,
-    },
-  ],
-  stages: [
-    {
-      id: 'hYbmMy3b7pz7GLiaT',
-      name: 'oneByTwo',
-      viewportStructure: {
-        type: 'grid',
-        properties: {
-          rows: 2,
-          columns: 2,
-        },
-      },
-      viewports: [
-        {
-          viewportSettings: [
-            {
-              options: {
-                voi: {
-                  windowWidth: 500,
-                  windowCenter: 500,
-                },
+const hangingProtocolName = 'default';
+
+/**
+ * Define some hanging protocols that can be activated by setting the priority
+ * See hangingProtocols.json5 for examples.
+ */
+const hangingProtocolConfig = ConfigPoint.createConfiguration("hangingProtocolConfig", {
```

Comment:
This defines four different hanging protocols that can be activated - see the hangingProtocols.json5 file for an example.